### PR TITLE
Backend/feature/community api

### PR DESCRIPTION
**Pull request by amdwit**

### CODE REVIEW

File: app/pb/community.proto [hunk 1]
```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
+
+package org.couchers.community;
+import "google/protobuf/timestamp.proto";
+
+import "pb/nullables.proto";
+
+message Point{
+  float lat = 1;
+  float lng = 2;
```
Comment (aapeliv) [inline app/pb/community.proto:10]: These need to be doubles

Comment (kalvdans) [reply app/pb/community.proto:10]: Do we need more than 4 meters resolution?

File: app/backend/src/couchers/servicers/pages.py [hunk 2]
```diff
@@ -16,8 +16,8 @@ def CreatePage(self, request: pages_pb2.CreatePageReq, context):
     def EditPage(self, request: pages_pb2.EditPageReq, context):
         with session_scope() as session:
             page = session.query(Page).get(request.page_id)
-            if page.can_edit(context.user_id):
+            try:
                 page.edit(context.user_id, **request)
-            else:
+            except:
```
Comment (kalvdans) [inline app/backend/src/couchers/servicers/pages.py]: Please state which exception you catch, instead of caching all errors including KeyboardInterrupt.

Comment (amdwit) [reply app/backend/src/couchers/servicers/pages.py]: Yes, I will do. Maybe we need to define some special exception(s)

File: app/pb/community.proto (hunk 1 above)
Comment (aapeliv) [reply app/pb/community.proto:10]: Yes. There's also no reason to use floats instead

**Comment by aapeliv** [comment]
* Why are there two (almost?) identical proto files? (`pages.proto` and `community.proto`)
* I still don't see the need for the `Nullable*Value`s in these protos

Please try to mimic the style of the rest of the code, both in terms of formatting but also in how stuff is laid out.

* The business logic shouldn't be in the models, it should be in the servicer.
* Please look at how `.proto` files are formatted (`service` at the top of file, careful with whitespace, etc). I can look for a formatter if you feel this would be worth it, for now it's been okay to do them by hand.
* The `page_session` fixture should be consistently named and be next to the other `*_session`s
* Files should end in a newline

**Pull request closed without merging**